automatic/autohotkey.install/tools/chocolateyInstall.ps1

```diff
@@ -1,9 +1,34 @@
-﻿$name   = 'autohotkey.install'
-$url = '{{DownloadUrl}}'
-$url64 = $url
+﻿$ErrorActionPreference = 'Stop'
+
+$packageName = 'autohotkey.install'
+$url32       = 'https://autohotkey.com/download/1.1/AutoHotkey112401_Install.exe'
+$url64       = $url32
+$checksum32  = '78ea671c3305465933a5d00f39da7e86c116a8a5a1849835196a27799ac29f8f'
+$checksum64  = $checksum32
+
 $silent = '/S'
+if(Get-ProcessorBits 64) { $silent += ' /x64' }
+
+$packageArgs = @{
+  packageName            = $packageName
+  fileType               = 'EXE'
+  url                    = $url32
+  url64bit               = $url64
+  checksum               = $checksum32
+  checksum64             = $checksum64
+  checksumType           = 'sha256'
+  checksumType64         = 'sha256'
+  silentArgs             = $silent
+  validExitCodes         = @(0)
+  registryUninstallerKey = 'autohotkey'
+}
+Install-ChocolateyPackage @packageArgs
 
-$is64bit = (Get-WmiObject Win32_Processor).AddressWidth -eq 64
-if($is64bit) { $silent += ' /x64' }
+$installLocation = Get-AppInstallLocation $packageArgs.registryUninstallerKey
+if ($installLocation)  {
+    Write-Host "$packageName installed to '$installLocation'"

```

Comment:
It definitely feels like this logic should be in core Chocolatey. It already has some of this, it would not be a stretch to add the search in Porgram Files like you are doing.